mithril-client/tests/extensions/fake_aggregator/mod.rs

```diff
@@ -6,6 +6,10 @@ mod cardano_transaction;
 mod certificate;
 mod mithril_stake_distribution;
 
+#[cfg(all(feature = "fs", feature = "unstable"))]
+#[allow(unused_imports)]
```

Comment:
Is it really necessary?

Reply:
Yes, else a "Unused import" clippy warning is raised when compiling integration test that does not use this structure (since they are separated binaries).